api/dfx/hooks/auth.hook.ts

```diff
@@ -10,9 +10,10 @@ export interface AuthInterface {
 
 export function useAuth(): AuthInterface {
   const { call } = useApi();
+  const message = 'By_signing_this_message,_you_confirm_that_you_are_the_sole_owner_of_the_provided_Blockchain_address._Your_ID:_';
 
   async function getSignMessage(address: string): Promise<string> {
-    return await call<SignMessage>({ url: `${AuthUrl.signMessage}?address=${address}`, method: 'GET' }).then(result => result.message);
+    return Promise.resolve(`${message}${address}`);
```

Comment:
@davidleomay hopefully it makes more sense now. I will remove the promise on #43, don't want to it in this PR to avoid touching other files that expect `getSignMessage` to return a Promise<string>

Reply:
Ok, now I see your problem, makes sense 👍